src/clouds.cpp

```diff
@@ -58,13 +57,13 @@ Clouds::Clouds(scene::ISceneManager* mgr,
 	//m_material.MaterialType = video::EMT_TRANSPARENT_VERTEX_ALPHA;
 	m_material.MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
 
+	m_params.height        = cloudheight ? cloudheight : 120;
```

Comment:
Got it: the [guiEngine](https://github.com/minetest/minetest/blob/master/src/guiEngine.cpp#L314) (whatever that is? I don't know all the minetest corners yet…) calls it with a value of 100.